components/brave_wallet/browser/internal/hd_key.cc

```diff
@@ -36,6 +37,22 @@ constexpr size_t kSHA512Length = 64;
 constexpr uint32_t kHardenedOffset = 0x80000000;
 constexpr size_t kSerializationLength = 78;
 constexpr size_t kMaxDerSignatureSize = 72;
+constexpr size_t kContextRandomizeSize = 32;
+
+const secp256k1_context* GetSecp256k1Ctx() {
+  static const secp256k1_context* const kSecp256k1Ctx = [] {
+    secp256k1_context* context = secp256k1_context_create(
+        SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY);
+
+    uint8_t bytes[kContextRandomizeSize] = {};
```

Comment:
please use `using SecureVector = std::vector<uint8_t, SecureZeroAllocator<uint8_t>>` for temporary random bytes